adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantAnnotationConverter.scala

```diff
@@ -113,7 +113,7 @@ object VariantAnnotationConverter extends Serializable {
   private def createFieldMap(keys: Seq[AttrKey], schema: Schema): Map[String, (Int, Object => Object)] = {
     keys.filter(_.attrConverter != null).map(field => {
       val avroField = schema.getField(field.adamKey)
-      field.vcfKey -> (avroField.pos, field.attrConverter)
+      field.vcfKey -> ((avroField.pos, field.attrConverter))
```

Comment:
The compiler is in this case auto-tupling and inserting the parentheses that I'm here making more explicit. See e.g. http://stackoverflow.com/a/29252334/47978

It's an [`ill--documented`](https://issues.scala-lang.org/browse/SI-3583) and now kind of deprecated feature.